examples/Search_reranking_with_cross-encoders.ipynb

```diff
@@ -787,7 +787,7 @@
         "\n",
         "There is also a latency impact of using ```text-davinci-003``` that you'll need to consider, with even our few examples above taking a couple seconds each - again, the ```Fine-tuning``` endpoint may help you here if you are able to get decent results from an ```ada``` or ```babbage``` fine-tuned model.\n",
         "\n",
-        "We've used the ```Completions``` endpoint from OpenAI to build our cross-encoder, but this area is well-served by the open-source community. [Here](https://huggingface.co/cross-encoder/mmarco-mMiniLMv2-L12-H384-v1) is an example from HuggingFace, for example.\n",
```

Comment:
this page is giving 404

Reply:
The model is available here: https://huggingface.co/jeffwan/mmarco-mMiniLMv2-L12-H384-v1